orte/mca/ess/Makefile.am

```diff
@@ -17,7 +17,9 @@
 # $HEADER$
 #
 
-AM_CPPFLAGS = $(LTDLINCL)
+# Add unwind flags because all of the files in this tree are
+# involved in startup.
+AM_CPPFLAGS = $(LTDLINCL) $(MPIR_UNWIND_CFLAGS)
```

Comment:
CPPFLAGS are different than CFLAGS.  I think you want to set `AM_CFLAGS`.

Also, setting the value here in this `Makefile.am` will propagate it to `orte/mca/ess/base`, but not to the individual components.  You might need to put it in each of the ess components' `Makefile.am`s.  ☹️ 

I see you edited `pmi/Makefile.am` -- do you need to also do the others for completeness?

Reply:
I don't think this need be done elsewhere - only the ess/pmi component is used by application procs. The other components are solely used by daemons and thus are not accessible by the debugger.

Reply:
Ok, cool.  Might be worth a comment in the Makefile.am so that we don't have this conversation again in 6 months.